classes/AddressChecksumCore.php

```diff
@@ -49,7 +49,7 @@ public function generateChecksum($address)
         try {
             $fields = $address->getFields();
         } catch (Exception $e) {
-            throw new Exception($e->getMessage());
+            throw new Exception($this->trans('Your address is incomplete, please update it.', array(), 'Shop.Notifications.Error'));
```

Comment:
I tried to keep some coherence with other classes. Also, if anyone ever adds a function with translatable strings it will use the trans function directly.